### ISSUE TITLE
[Feature Request] Add PgUp/PgDown keybindings

### ISSUE THREAD

**Issue by BachoSeven** [ClementTsang/bottom#232]
## Describe the feature request

Hi, I think that it would be awesome to have `PgUp/PgDown` bindings to quickly move around the processes tab, and especially useful in `basic` mode.

## Additional context/details

It would ofc be awesome for these to also have correspondent`vi` bindings with `Ctrl+u` and `Ctrl+d`:) .

**Comment by BachoSeven** [comment]
@ClementTsang I noticed that in one of the latest releases support for `PgUp/PgDown` was added through https://github.com/ClementTsang/bottom/pull/646, so this is partially solved. Would still like tu have `Ctrl+u` and `Ctrl+d` bound to the same actions as well, though. Perhaps that can wait for https://github.com/ClementTsang/bottom/issues/454, though(?) in which case this could be closed, I guess.
